### DISCUSSION TITLE
Contact form only when logged in

### DISCUSSION THREAD

**Discussion by Derkades** [NamelessMC/Nameless#2173] [Completed Ideas]
Add option to only allow submitting contact form when logged in

**Comment by partydragen** [comment]
Using Forms Module as Contact form can pretty much do it all also going to make forms be able to send emails

**Comment by StormyIceLeopard** [comment]
How about instead an easy way to disable the Contact Us Button all together or have group permission control for it, like announcements and forums.

**Comment by BaxAndrei** [comment]
I think that an option to let you check if it is necessary to be logged in or not to use the contact form is enough.

**Reply by partydragen** [reply]
Just use Forms module instead then override contact form by making a form called /contact

**Reply by karkkijokeri** [reply]
partydragen, why do you want everyone to use your forms module if only thing they want to do is allow /contact for specific people 😆

**Comment by PikaMug** [comment]
I feel a combination of https://github.com/NamelessMC/Nameless/issues/2260 and https://github.com/NamelessMC/Nameless/discussions/2543 would be more versatile.

**Comment by supercrafter100** [comment]
The contact us form was completely removed in pr13. Use the forms module if you still want it
